# Project Introduction 

After the pandemic in 2020, Australia's house market surged instead of diving as initially expected by economists and analysts. Meanwhile, the apartment market suffered from migration loss but is recovering. No matter home buyers or property investors are keen to know how the market is going on in different suburbs, and where is the next opportunity.

Under this circumstance, the project aims to find out the target suburbs for potential investment opportunities in the residential property market of Australia’s biggest cities. As it is based on suburbs, the factor state is weakened in the report.

# Data Collection 

## Suburbs from Top 5 cities in population

The suburbs selected are in the top 5 cities in population in Australia: Sydney(4.8m), Melbourne(4.7m), Brisbane(2.3m), Perth(2.2m), and Adelaide(1.2m) by 2021. [^1]

In each city, the suburbs selected are by sequential postcodes from the CBD.

Total number of suburbs: 905
- Sydney: 322  (postcode: 2000-2155)
- Melbourne: 232  (postcode: 3000-3140)
- Brisbane: 148  (postcode: 4000-4110)
- Perth: 99  (postcode: 6000-6050)
- Adelaide: 104  (postcode: 5000-5040)

## Sample properties from Domain webiste

The sample properties are extracted from Domain's website by 23 Jul 2021, with 653,022 records in total.

It covers all the sold houses and apartments(incl. apartment, unit, flat) in history in each selected suburb. 

The properties with a sold price under $200,000 are removed as invalid data.

The suburbs with less than 30 sold properties in history is removed as small samples.

# Assumption

The report assumes that Domain’s website represents the residential property sales market in Australia. Domain is one of the two dominant players in the residential property advertising market in Australia (incl. online and offline). It has the most listings and potential customers.[^2]

The report takes the buyers' budget as the key factor for making property investment. The other influencing factors such as rental yield, train station, shopping centre, schools etc. are not included at this stage.

The report takes the distance to CBD as an important consideration. Though the covidvirus pandemic is influencing home buyers' decisions currently, the location still has its value in the long term from the author's view.

Investors are target clients for this report.

The median prices in 2021 is based on the sold properties by 23 Jul 2021.

# Technologies for Analytics

1. **Python**: for scraping, parsing, cleansing and loading data
2. **SQL**: for data querying in database, transforming and analysing
3. **Jupyter Notebook**: for data analysis and reporting
4. **PowerBI/Tableau**: for data visualisation and ad-hoc analysis
5. **Docker**: deploying analysing environment

For technical details please see this notebook: [domain.ipynb](https://github.com/helengtt/properties-investment/blob/main/ipynb/domain.ipynb)


# House Market Analysis

As of the pandemic from 2020, the House market and Apartment market are two different stories in Australia's residential property market. While house prices keep surging after the first lockdown, apartment prices bounced mildly after reaching the bottom. Therefore, the report will analyse the two markets separately for investors.

For house market, we'll take the following steps to shortern the suburbs list and assist investors to find out the target suburbs.

## Profile Assumption: House Investor with $2M Budget

Everyone knows picking up suitable suburbs from all the suburbs can be an overwhelming problem for all investors. Let's start with something every single investor would consider, which is budgeting. Different investors with different budgeting will have very different choices of suburbs. We assume a budget of $2M for the house investor here.

### 1. Suburbs' Affordability

Affordability means the percentage of houses sold in the price range to all those sold in the suburb.

The affordability here is based on the house sold in the past year because the price grew fast after 1 Jul 2020.

For the house investors, we select the house from \\$1.5M to \\$2M to meet their budget. 

In [2]:
# %load_ext sql
# %sql postgresql://postgres:password@this_postgres
%sql SELECT count(suburb_id) as suburbs, sum(ttl_house_sold) as total_house_sold, sum(ttl_house_tier2) as house_sold_tier2 from combined_suburbs

 * postgresql://postgres:***@this_postgres
1 rows affected.


[(905, Decimal('51044'), Decimal('6000'))]

We've found 6,000 out of 51,044 houses sold between \\$1.5M and \\$2M in the 905 selected suburbs. 

<img src='../charts/house_cnt_1yr.png'>

*Source: Domain*

The chart below illustrates the affordability under \\$1.5M-\\$2M budget.

<img src = '../charts/house_aff.png'>

*Source: Domain*

Next, we'll filter the suburbs in the following criteria:
- Affordability over 30%. From my personal perspective, it doesn't make sense to choose the suburbs where you can only buy less than 30% of the properties because you wouldn't have many choices.
- Minimum 10 houses were sold in the price range in the past year.

In [17]:
%sql SELECT suburb_id, ttl_house_tier2, house_affordability_tier2 from combined_suburbs where house_affordability_tier2 >= 0.3 and ttl_house_tier2 >=10 order by ttl_house_tier2 desc

 * postgresql://postgres:***@this_postgres
97 rows affected.


[('castle-hill-nsw-2154', 159, Decimal('0.473')),
 ('kellyville-nsw-2155', 111, Decimal('0.329')),
 ('baulkham-hills-nsw-2153', 103, Decimal('0.346')),
 ('wahroonga-nsw-2076', 80, Decimal('0.351')),
 ('leichhardt-nsw-2040', 78, Decimal('0.506')),
 ('ryde-nsw-2112', 76, Decimal('0.447')),
 ('newtown-nsw-2042', 74, Decimal('0.457')),
 ('frenchs-forest-nsw-2086', 73, Decimal('0.483')),
 ('carlingford-nsw-2118', 73, Decimal('0.437')),
 ('cherrybrook-nsw-2126', 72, Decimal('0.545')),
 ('epping-nsw-2121', 69, Decimal('0.423')),
 ('doncaster-east-vic-3109', 66, Decimal('0.316')),
 ('west-pennant-hills-nsw-2125', 62, Decimal('0.332')),
 ('northcote-vic-3070', 59, Decimal('0.428')),
 ('stanmore-nsw-2048', 52, Decimal('0.536')),
 ('north-ryde-nsw-2113', 52, Decimal('0.500')),
 ('eastwood-nsw-2122', 49, Decimal('0.380')),
 ('templestowe-vic-3106', 47, Decimal('0.326')),
 ('essendon-vic-3040', 46, Decimal('0.351')),
 ('hornsby-nsw-2077', 46, Decimal('0.393')),
 ('rozelle-nsw-2039', 44, Decimal('0.358')),
 ('doncaster-vic-3108', 42, Decimal('0.321')),
 ('west-ryde-nsw-2114', 42, Decimal('0.592')),
 ('belrose-nsw-2085', 41, Decimal('0.436')),
 ('balwyn-north-vic-3104', 41, Decimal('0.373')),
 ('beecroft-nsw-2119', 40, Decimal('0.421')),
 ('beacon-hill-nsw-2100', 39, Decimal('0.500')),
 ('lilyfield-nsw-2040', 37, Decimal('0.327')),
 ('forestville-nsw-2087', 37, Decimal('0.440')),
 ('bella-vista-nsw-2153', 36, Decimal('0.364')),
 ('thornleigh-nsw-2120', 36, Decimal('0.379')),
 ('blackburn-vic-3130', 35, Decimal('0.318')),
 ('north-rocks-nsw-2151', 34, Decimal('0.400')),
 ('north-epping-nsw-2121', 34, Decimal('0.607')),
 ('pennant-hills-nsw-2120', 33, Decimal('0.458')),
 ('ermington-nsw-2115', 32, Decimal('0.317')),
 ('cromer-nsw-2099', 32, Decimal('0.444')),
 ('glebe-nsw-2037', 32, Decimal('0.478')),
 ('warriewood-nsw-2102', 29, Decimal('0.492')),
 ('gladesville-nsw-2111', 29, Decimal('0.363')),
 ('matraville-nsw-2036', 29, Decimal('0.509')),
 ('ashfield-nsw-2131', 29, Decimal('0.433')),
 ('thornbury-vic-3071', 28, Decimal('0.308')),
 ('north-narrabeen-nsw-2101', 28, Decimal('0.424')),
 ('strathmore-vic-3041', 28, Decimal('0.359')),
 ('allambie-heights-nsw-2100', 27, Decimal('0.375')),
 ('mascot-nsw-2020', 26, Decimal('0.473')),
 ('croydon-park-nsw-2133', 25, Decimal('0.463')),
 ('enmore-nsw-2042', 25, Decimal('0.568')),
 ('normanhurst-nsw-2076', 25, Decimal('0.463')),
 ('petersham-nsw-2049', 25, Decimal('0.490')),
 ('surrey-hills-vic-3127', 25, Decimal('0.446')),
 ('erskineville-nsw-2043', 24, Decimal('0.333')),
 ('croydon-nsw-2132', 24, Decimal('0.421')),
 ('alexandria-nsw-2015', 23, Decimal('0.397')),
 ('box-hill-vic-3128', 23, Decimal('0.479')),
 ('camperdown-nsw-2050', 22, Decimal('0.564')),
 ('ivanhoe-vic-3079', 22, Decimal('0.379')),
 ('five-dock-nsw-2046', 22, Decimal('0.386')),
 ('park-orchards-vic-3114', 20, Decimal('0.400')),
 ('denistone-nsw-2114', 20, Decimal('0.500')),
 ('hawthorn-vic-3122', 20, Decimal('0.417')),
 ('st-lucia-qld-4067', 20, Decimal('0.377')),
 ('new-farm-qld-4005', 20, Decimal('0.333')),
 ('st-peters-nsw-2044', 20, Decimal('0.435')),
 ('telopea-nsw-2117', 19, Decimal('0.594')),
 ('kew-east-vic-3102', 19, Decimal('0.543')),
 ('marsfield-nsw-2122', 19, Decimal('0.404')),
 ('brunswick-east-vic-3057', 18, Decimal('0.409')),
 ('west-pymble-nsw-2073', 18, Decimal('0.367')),
 ('surry-hills-nsw-2010', 18, Decimal('0.327')),
 ('wheeler-heights-nsw-2097', 18, Decimal('0.581')),
 ('east-ryde-nsw-2113', 18, Decimal('0.621')),
 ('dee-why-nsw-2099', 17, Decimal('0.333')),
 ('collaroy-plateau-nsw-2097', 17, Decimal('0.378')),
 ('summer-hill-nsw-2130', 16, Decimal('0.364')),
 ('narraweena-nsw-2099', 16, Decimal('0.432')),
 ('botany-nsw-2019', 16, Decimal('0.444')),
 ('killarney-heights-nsw-2087', 15, Decimal('0.357')),
 ('clifton-hill-vic-3068', 15, Decimal('0.441')),
 ('floreat-wa-6014', 14, Decimal('0.341')),
 ('elanora-heights-nsw-2101', 14, Decimal('0.350')),
 ('westmead-nsw-2145', 14, Decimal('0.737')),
 ('nedlands-wa-6009

We get 97 suburbs, each of which has at least 30% houses affordable. 

### 2. Distance to CBD

Now we'll take the distance to CBD as another criterion to filter the suburbs.

<img src = '../charts/house_dis.png'>

<img src = '../charts/house_dis2.png'>

*Source: Domain, World Bank*

We take 12km from CBD as a reasonable measure at this stage.

In [57]:
%%sql 
select suburb_id, ttl_house_tier2, house_affordability_tier2, distance_km
from combined_suburbs
where house_affordability_tier2 >= 0.3 and ttl_house_tier2 >=10 and distance_km <= 15
order by distance_km

We get 65 suburbs within 20km in total.

### 3. Investment by Median Price Growth

Usually, it is reasonable for property investors to expect a minimum 30% growth in 5 years. So we shorten the suburb list with this criteria.

In [14]:
%%sql
select suburb_id, house_5yr_growth, house_2017_growth, house_2018_growth, house_2019_growth, house_2020_growth, house_2021_growth, price_house_2016, price_house_2017, price_house_2018, price_house_2019, price_house_2020, price_house_2021, ttl_house_tier2, house_affordability_tier2, distance_km
from combined_suburbs
where house_affordability_tier2 >= 0.3 and ttl_house_tier2 >=10 and distance_km <= 15 and house_5yr_growth >= 0.3
order by house_5yr_growth desc

 * postgresql://postgres:***@this_postgres
27 rows affected.


[('city-beach-wa-6015', Decimal('0.892'), Decimal('0.559'), Decimal('-0.061'), Decimal('-0.077'), Decimal('0.107'), Decimal('0.265'), Decimal('1110000'), Decimal('1730000'), Decimal('1625000'), Decimal('1500000'), Decimal('1660000'), Decimal('2100000'), 11, Decimal('0.324'), Decimal('8.874')),
 ('allambie-heights-nsw-2100', Decimal('0.680'), Decimal('0.127'), Decimal('-0.024'), Decimal('-0.027'), Decimal('0.125'), Decimal('0.395'), Decimal('1500000'), Decimal('1690000'), Decimal('1650000'), Decimal('1605000'), Decimal('1806000'), Decimal('2520000'), 27, Decimal('0.375'), Decimal('11.469')),
 ('st-lucia-qld-4067', Decimal('0.632'), Decimal('0.274'), Decimal('-0.116'), Decimal('0.051'), Decimal('0.391'), Decimal('-0.010'), Decimal('950000'), Decimal('1210000'), Decimal('1070000'), Decimal('1125000'), Decimal('1565000'), Decimal('1550000'), 20, Decimal('0.377'), Decimal('3.806')),
 ('killarney-heights-nsw-2087', Decimal('0.588'), Decimal('0.109'), Decimal('-0.162'), Decimal('0.114'), Decimal('0.122'), Decimal('0.367'), Decimal('1700000'), Decimal('1885000'), Decimal('1580000'), Decimal('1760000'), Decimal('1975000'), Decimal('2700000'), 15, Decimal('0.357'), Decimal('10.662')),
 ('floreat-wa-6014', Decimal('0.538'), Decimal('0.172'), Decimal('0.059'), Decimal('-0.050'), Decimal('0.211'), Decimal('0.077'), Decimal('1086000'), Decimal('1273000'), Decimal('1347500'), Decimal('1280000'), Decimal('1550000'), Decimal('1670000'), 14, Decimal('0.341'), Decimal('6.724')),
 ('narraweena-nsw-2099', Decimal('0.538'), Decimal('0.104'), Decimal('-0.119'), Decimal('0.014'), Decimal('0.092'), Decimal('0.429'), Decimal('1440000'), Decimal('1590000'), Decimal('1400000'), Decimal('1420000'), Decimal('1550000'), Decimal('2215000'), 16, Decimal('0.432'), Decimal('14.326')),
 ('forestville-nsw-2087', Decimal('0.505'), Decimal('0.197'), Decimal('-0.171'), Decimal('0.043'), Decimal('0.125'), Decimal('0.293'), Decimal('1462000'), Decimal('1750000'), Decimal('1450000'), Decimal('1511800'), Decimal('1701000'), Decimal('2200000'), 37, Decimal('0.440'), Decimal('11.811')),
 ('west-pymble-nsw-2073', Decimal('0.492'), Decimal('0.127'), Decimal('-0.103'), Decimal('-0.006'), Decimal('0.133'), Decimal('0.310'), Decimal('1642000'), Decimal('1850000'), Decimal('1660000'), Decimal('1650000'), Decimal('1870000'), Decimal('2450000'), 18, Decimal('0.367'), Decimal('13.751')),
 ('beacon-hill-nsw-2100', Decimal('0.483'), Decimal('0.167'), Decimal('-0.080'), Decimal('-0.066'), Decimal('0.152'), Decimal('0.283'), Decimal('1376000'), Decimal('1606000'), Decimal('1477000'), Decimal('1380000'), Decimal('1590000'), Decimal('2040000'), 39, Decimal('0.500'), Decimal('13.632')),
 ('concord-west-nsw-2138', Decimal('0.479'), Decimal('0.264'), Decimal('-0.163'), Decimal('-0.044'), Decimal('0.174'), Decimal('0.246'), Decimal('1701000'), Decimal('2150000'), Decimal('1800000'), Decimal('1721000'), Decimal('2020000'), Decimal('2516000'), 11, Decimal('0.306'), Decimal('11.228')),
 ('marsfield-nsw-2122', Decimal('0.442'), Decimal('0.043'), Decimal('-0.071'), Decimal('-0.047'), Decimal('0.163'), Decimal('0.343'), Decimal('1630000'), Decimal('1700000'), Decimal('1580000'), Decimal('1505000'), Decimal('1750000'), Decimal('2350000'), 19, Decimal('0.404'), Decimal('13.963')),
 ('north-ryde-nsw-2113', Decimal('0.433'), Decimal('0.155'), Decimal('-0.134'), Decimal('0.013'), Decimal('0.079'), Decimal('0.311'), Decimal('1500000'), Decimal('1732500'), Decimal('1501000'), Decimal('1520000'), Decimal('1640000'), Decimal('2150000'), 52, Decimal('0.500'), Decimal('10.630')),
 ('ryde-nsw-2112', Decimal('0.400'), Decimal('0.167'), Decimal('-0.137'), Decimal('-0.030'), Decimal('0.154'), Decimal('0.242'), Decimal('1500000'), Decimal('1751000'), Decimal('1510500'), Decimal('1465000'), Decimal('1691000'), Decimal('2100000'), 76, Decimal('0.447'), Decimal('11.105')),
 ('dee-why-nsw-2099', Decimal('0.390'), Decimal('0.053'), Decimal('-0.164'), Decimal('0.163'), Decimal('0.060'), Decimal('0.281'), Decimal('1705

Now we get 27 suburbs.

Next, we need to do some risk control. We know Australia's house prices have experienced the largest downturn on record between July 2017 and May 2019 as of the tightened lending conditions and changes of loan limits to overseas investors by the government. Investors would avoid loseing money while seeking a high ROI suburb for investment. Thus we remove the suburbs with a decrease over 10% in 2018 and 2019.

In [18]:
%%sql 
select suburb_id, house_5yr_growth, house_2017_growth, house_2018_growth, house_2019_growth, house_2020_growth, house_2021_growth, price_house_2016, price_house_2017, price_house_2018, price_house_2019, price_house_2020, price_house_2021, ttl_house_tier2, house_affordability_tier2, distance_km
from combined_suburbs
where house_affordability_tier2 >= 0.3 and ttl_house_tier2 >=10 and distance_km <= 15 and house_5yr_growth >= 0.3 and house_2019_growth >= -0.1 and house_2018_growth >= -0.1
order by house_5yr_growth desc

 * postgresql://postgres:***@this_postgres
12 rows affected.


[('city-beach-wa-6015', Decimal('0.892'), Decimal('0.559'), Decimal('-0.061'), Decimal('-0.077'), Decimal('0.107'), Decimal('0.265'), Decimal('1110000'), Decimal('1730000'), Decimal('1625000'), Decimal('1500000'), Decimal('1660000'), Decimal('2100000'), 11, Decimal('0.324'), Decimal('8.874')),
 ('allambie-heights-nsw-2100', Decimal('0.680'), Decimal('0.127'), Decimal('-0.024'), Decimal('-0.027'), Decimal('0.125'), Decimal('0.395'), Decimal('1500000'), Decimal('1690000'), Decimal('1650000'), Decimal('1605000'), Decimal('1806000'), Decimal('2520000'), 27, Decimal('0.375'), Decimal('11.469')),
 ('floreat-wa-6014', Decimal('0.538'), Decimal('0.172'), Decimal('0.059'), Decimal('-0.050'), Decimal('0.211'), Decimal('0.077'), Decimal('1086000'), Decimal('1273000'), Decimal('1347500'), Decimal('1280000'), Decimal('1550000'), Decimal('1670000'), 14, Decimal('0.341'), Decimal('6.724')),
 ('beacon-hill-nsw-2100', Decimal('0.483'), Decimal('0.167'), Decimal('-0.080'), Decimal('-0.066'), Decimal('0.152'), Decimal('0.283'), Decimal('1376000'), Decimal('1606000'), Decimal('1477000'), Decimal('1380000'), Decimal('1590000'), Decimal('2040000'), 39, Decimal('0.500'), Decimal('13.632')),
 ('marsfield-nsw-2122', Decimal('0.442'), Decimal('0.043'), Decimal('-0.071'), Decimal('-0.047'), Decimal('0.163'), Decimal('0.343'), Decimal('1630000'), Decimal('1700000'), Decimal('1580000'), Decimal('1505000'), Decimal('1750000'), Decimal('2350000'), 19, Decimal('0.404'), Decimal('13.963')),
 ('st-peters-nsw-2044', Decimal('0.376'), Decimal('-0.008'), Decimal('0.040'), Decimal('0.044'), Decimal('-0.021'), Decimal('0.306'), Decimal('1210000'), Decimal('1200000'), Decimal('1247500'), Decimal('1302000'), Decimal('1275000'), Decimal('1665000'), 20, Decimal('0.435'), Decimal('6.157')),
 ('enmore-nsw-2042', Decimal('0.369'), Decimal('0.043'), Decimal('-0.039'), Decimal('0.007'), Decimal('0.185'), Decimal('0.144'), Decimal('1337000'), Decimal('1395000'), Decimal('1340000'), Decimal('1350000'), Decimal('1600000'), Decimal('1830000'), 25, Decimal('0.568'), Decimal('5.159')),
 ('croydon-park-nsw-2133', Decimal('0.360'), Decimal('0.172'), Decimal('-0.094'), Decimal('-0.003'), Decimal('0.030'), Decimal('0.247'), Decimal('1327000'), Decimal('1555000'), Decimal('1409000'), Decimal('1405000'), Decimal('1447000'), Decimal('1805000'), 25, Decimal('0.463'), Decimal('10.412')),
 ('matraville-nsw-2036', Decimal('0.353'), Decimal('0.062'), Decimal('-0.061'), Decimal('-0.044'), Decimal('0.080'), Decimal('0.314'), Decimal('1700000'), Decimal('1805000'), Decimal('1695000'), Decimal('1621000'), Decimal('1750000'), Decimal('2300000'), 29, Decimal('0.509'), Decimal('10.901')),
 ('new-farm-qld-4005', Decimal('0.342'), Decimal('-0.161'), Decimal('0.157'), Decimal('-0.008'), Decimal('0.187'), Decimal('0.174'), Decimal('1610000'), Decimal('1350000'), Decimal('1562500'), Decimal('1550000'), Decimal('1840000'), Decimal('2160000'), 20, Decimal('0.333'), Decimal('2.305')),
 ('northcote-vic-3070', Decimal('0.312'), Decimal('0.087'), Decimal('0.019'), Decimal('-0.015'), Decimal('0.120'), Decimal('0.073'), Decimal('1218500'), Decimal('1325000'), Decimal('1350000'), Decimal('1330000'), Decimal('1490000'), Decimal('1599000'), 59, Decimal('0.428'), Decimal('6.609')),
 ('alexandria-nsw-2015', Decimal('0.301'), Decimal('0.092'), Decimal('-0.060'), Decimal('0.033'), Decimal('0.200'), Decimal('0.022'), Decimal('1460000'), Decimal('1595000'), Decimal('1500000'), Decimal('1550000'), Decimal('1860000'), Decimal('1900000'), 23, Decimal('0.397'), Decimal('4.777'))]

Now 12 suburbs are left for the investors from 905 suburbs in the top 5 cities. 

There are 8 out of 12 suburbs from NSW, 2 from West Australia, 1 from Victoria and 1 from Queensland.

<img src='../charts/house_result.png'>

<img src = '../charts/house_result2.png'>

*Source: Domain, World Bank*

In [ ]:
line chart: one suburb which covers medium price from 2010 to 2021, 找曲线最差的拿出来for example，说明风控必要性。
    map chart：

**Result Table**

In [22]:
%%sql
select c.suburb_id, s.suburb, s.state, s.population, s.median_income
    , house_5yr_growth, house_2017_growth, house_2018_growth, house_2019_growth, house_2020_growth, house_2021_growth, price_house_2016, price_house_2017, price_house_2018, price_house_2019, price_house_2020, price_house_2021, ttl_house_tier2, house_affordability_tier2, distance_km
from combined_suburbs c
left join suburbs s on c.suburb_id = s.suburb_id
where house_affordability_tier2 >= 0.3 and ttl_house_tier2 >=10 and distance_km <= 15 and house_5yr_growth >= 0.3 and house_2019_growth >= -0.1 and house_2018_growth >= -0.1
order by house_5yr_growth desc

 * postgresql://postgres:***@this_postgres
12 rows affected.


[('city-beach-wa-6015', 'City Beach', 'WA', 6609, 53196, Decimal('0.892'), Decimal('0.559'), Decimal('-0.061'), Decimal('-0.077'), Decimal('0.107'), Decimal('0.265'), Decimal('1110000'), Decimal('1730000'), Decimal('1625000'), Decimal('1500000'), Decimal('1660000'), Decimal('2100000'), 11, Decimal('0.324'), Decimal('8.874')),
 ('allambie-heights-nsw-2100', 'Allambie Heights', 'NSW', 7009, 40404, Decimal('0.680'), Decimal('0.127'), Decimal('-0.024'), Decimal('-0.027'), Decimal('0.125'), Decimal('0.395'), Decimal('1500000'), Decimal('1690000'), Decimal('1650000'), Decimal('1605000'), Decimal('1806000'), Decimal('2520000'), 27, Decimal('0.375'), Decimal('11.469')),
 ('floreat-wa-6014', 'Floreat', 'WA', 7899, 54184, Decimal('0.538'), Decimal('0.172'), Decimal('0.059'), Decimal('-0.050'), Decimal('0.211'), Decimal('0.077'), Decimal('1086000'), Decimal('1273000'), Decimal('1347500'), Decimal('1280000'), Decimal('1550000'), Decimal('1670000'), 14, Decimal('0.341'), Decimal('6.724')),
 ('beacon-hill-nsw-2100', 'Beacon Hill', 'NSW', 7457, 41288, Decimal('0.483'), Decimal('0.167'), Decimal('-0.080'), Decimal('-0.066'), Decimal('0.152'), Decimal('0.283'), Decimal('1376000'), Decimal('1606000'), Decimal('1477000'), Decimal('1380000'), Decimal('1590000'), Decimal('2040000'), 39, Decimal('0.500'), Decimal('13.632')),
 ('marsfield-nsw-2122', 'Marsfield', 'NSW', 13297, 32448, Decimal('0.442'), Decimal('0.043'), Decimal('-0.071'), Decimal('-0.047'), Decimal('0.163'), Decimal('0.343'), Decimal('1630000'), Decimal('1700000'), Decimal('1580000'), Decimal('1505000'), Decimal('1750000'), Decimal('2350000'), 19, Decimal('0.404'), Decimal('13.963')),
 ('st-peters-nsw-2044', 'St Peters', 'NSW', 3145, 58968, Decimal('0.376'), Decimal('-0.008'), Decimal('0.040'), Decimal('0.044'), Decimal('-0.021'), Decimal('0.306'), Decimal('1210000'), Decimal('1200000'), Decimal('1247500'), Decimal('1302000'), Decimal('1275000'), Decimal('1665000'), 20, Decimal('0.435'), Decimal('6.157')),
 ('enmore-nsw-2042', 'Enmore', 'NSW', 3880, 53872, Decimal('0.369'), Decimal('0.043'), Decimal('-0.039'), Decimal('0.007'), Decimal('0.185'), Decimal('0.144'), Decimal('1337000'), Decimal('1395000'), Decimal('1340000'), Decimal('1350000'), Decimal('1600000'), Decimal('1830000'), 25, Decimal('0.568'), Decimal('5.159')),
 ('croydon-park-nsw-2133', 'Croydon Park', 'NSW', 11012, 35360, Decimal('0.360'), Decimal('0.172'), Decimal('-0.094'), Decimal('-0.003'), Decimal('0.030'), Decimal('0.247'), Decimal('1327000'), Decimal('1555000'), Decimal('1409000'), Decimal('1405000'), Decimal('1447000'), Decimal('1805000'), 25, Decimal('0.463'), Decimal('10.412')),
 ('matraville-nsw-2036', 'Matraville', 'NSW', 9804, 36452, Decimal('0.353'), Decimal('0.062'), Decimal('-0.061'), Decimal('-0.044'), Decimal('0.080'), Decimal('0.314'), Decimal('1700000'), Decimal('1805000'), Decimal('1695000'), Decimal('1621000'), Decimal('1750000'), Decimal('2300000'), 29, Decimal('0.509'), Decimal('10.901')),
 ('new-farm-qld-4005', 'New Farm', 'QLD', 12542, 55172, Decimal('0.342'), Decimal('-0.161'), Decimal('0.157'), Decimal('-0.008'), Decimal('0.187'), Decimal('0.174'), Decimal('1610000'), Decimal('1350000'), Decimal('1562500'), Decimal('1550000'), Decimal('1840000'), Decimal('2160000'), 20, Decimal('0.333'), Decimal('2.305')),
 ('northcote-vic-3070', 'Northcote', 'VIC', 24561, 45708, Decimal('0.312'), Decimal('0.087'), Decimal('0.019'), Decimal('-0.015'), Decimal('0.120'), Decimal('0.073'), Decimal('1218500'), Decimal('1325000'), Decimal('1350000'), Decimal('1330000'), Decimal('1490000'), Decimal('1599000'), 59, Decimal('0.428'), Decimal('6.609')),
 ('alexandria-nsw-2015', 'Alexandria', 'NSW', 8262, 69732, Decimal('0.301'), Decimal('0.092'), Decimal('-0.060'), Decimal('0.033'), Decimal('0.200'), Decimal('0.022'), Decimal('1460000'), Decimal('1595000'), Decimal('1500000'), Decimal('1550000'), Decimal('1860000'), Decimal('1900000'), 23, Decimal('0.397'), Decimal('4.777'))]

## Conclusion

The example successfully picked up 12 suburbs. However, this is just an example. We still have more features like percentage by bedrooms, the population, media income, monthly growth etc.. Anyone who knows SQL querying can adjust the parameters above to select their desiable suburb(s). Then the investors can decide in which suburbs they'll inspect properties. If we enrich the dataset with other relevant data, we would be able to add more value.

# Apartment Market Analysis

The apartment market has experienced a heavy strike after the pandemic, in contrast to the white-hot house market. However, family-friendly apartments in medium and low rise buildings still have their value as an affordable alternative to houses in popular areas. With capital growth in houses outperforming apartments so far this year, the increasing demand for these more affordable apartments are moving forward. 

For investors, the apartment price is at its low level the moment. We could find some cost-effective opportunities if we select carefully and patiently.

## Profile Assumption: Apartment Investor with $800K Budget

We assume a budget of $800K for the apartment investor here. Let's look at the affordability first.

### 1. Suburbs' Affordability

For the apartment investors, we select the apartments from \\$500K to \\$800K to meet their budget. The affordability is based on the apartments sold in the past year.

In [6]:
# %sql SELECT count(suburb_id) as suburbs, sum(ttl_apt_sold) as total_apt_sold, sum(ttl_apt_tier1) as apt_sold_tier1 from combined_suburbs
aff_apt = ('SELECT count(suburb_id) as suburbs, sum(ttl_apt_sold) as total_apt_sold, sum(ttl_apt_tier1) as apt_sold_tier1 from combined_suburbs', engine)


 * postgresql://postgres:***@this_postgres
1 rows affected.


[(905, Decimal('26256'), Decimal('9959'))]

We've found 9,959 out of 26,256 apartments sold in \\$500K - \\$800K in the 905 selected suburbs.

Now lets look into the affordability.

<img src='../charts/apt_aff.png'>

*Source: Domain*

Next, we'll filter the suburbs in the following criteria:

- Affordability over 30%.
- Minimum 20 apartments were sold in the price range in the past year.

In [7]:
%sql SELECT suburb_id, ttl_apt_tier1, apt_affordability_tier1 from combined_suburbs where apt_affordability_tier1 >= 0.3 and ttl_apt_tier1 >=20 order by ttl_apt_tier1 desc

 * postgresql://postgres:***@this_postgres
123 rows affected.


[('wentworth-point-nsw-2127', 249, Decimal('0.819')),
 ('melbourne-vic-3000', 227, Decimal('0.374')),
 ('southbank-vic-3006', 174, Decimal('0.489')),
 ('ashfield-nsw-2131', 163, Decimal('0.674')),
 ('hornsby-nsw-2077', 162, Decimal('0.876')),
 ('parramatta-nsw-2150', 157, Decimal('0.623')),
 ('dee-why-nsw-2099', 143, Decimal('0.379')),
 ('lane-cove-nsw-2066', 137, Decimal('0.489')),
 ('ryde-nsw-2112', 123, Decimal('0.615')),
 ('westmead-nsw-2145', 116, Decimal('0.811')),
 ('docklands-vic-3008', 115, Decimal('0.458')),
 ('richmond-vic-3121', 115, Decimal('0.504')),
 ('reservoir-vic-3073', 111, Decimal('0.603')),
 ('strathfield-nsw-2135', 108, Decimal('0.720')),
 ('homebush-west-nsw-2140', 102, Decimal('0.887')),
 ('homebush-nsw-2140', 96, Decimal('0.842')),
 ('brunswick-vic-3056', 94, Decimal('0.534')),
 ('waterloo-nsw-2017', 94, Decimal('0.465')),
 ('epping-nsw-2121', 94, Decimal('0.648')),
 ('waitara-nsw-2077', 90, Decimal('0.882')),
 ('croydon-vic-3136', 87, Decimal('0.674')),
 ('brisbane-city-qld-4000', 86, Decimal('0.305')),
 ('hawthorn-vic-3122', 85, Decimal('0.515')),
 ('carlingford-nsw-2118', 85, Decimal('0.904')),
 ('west-end-qld-4101', 83, Decimal('0.432')),
 ('gladesville-nsw-2111', 79, Decimal('0.725')),
 ('ringwood-vic-3134', 76, Decimal('0.691')),
 ('north-parramatta-nsw-2151', 76, Decimal('0.724')),
 ('alexandria-nsw-2015', 74, Decimal('0.510')),
 ('brunswick-east-vic-3057', 71, Decimal('0.617')),
 ('teneriffe-qld-4005', 68, Decimal('0.557')),
 ('preston-vic-3072', 66, Decimal('0.458')),
 ('zetland-nsw-2017', 64, Decimal('0.403')),
 ('baulkham-hills-nsw-2153', 63, Decimal('0.863')),
 ('north-melbourne-vic-3051', 62, Decimal('0.596')),
 ('new-farm-qld-4005', 62, Decimal('0.373')),
 ('west-ryde-nsw-2114', 62, Decimal('0.775')),
 ('macquarie-park-nsw-2113', 59, Decimal('0.663')),
 ('glenroy-vic-3046', 58, Decimal('0.690')),
 ('essendon-vic-3040', 57, Decimal('0.460')),
 ('collingwood-vic-3066', 57, Decimal('0.514')),
 ('northcote-vic-3070', 56, Decimal('0.523')),
 ('rosebery-nsw-2018', 56, Decimal('0.459')),
 ('rhodes-nsw-2138', 54, Decimal('0.370')),
 ('newstead-qld-4006', 54, Decimal('0.478')),
 ('meadowbank-nsw-2114', 53, Decimal('0.697')),
 ('hawthorn-east-vic-3123', 53, Decimal('0.631')),
 ('altona-vic-3018', 52, Decimal('0.675')),
 ('mascot-nsw-2020', 48, Decimal('0.505')),
 ('brunswick-west-vic-3055', 47, Decimal('0.475')),
 ('maroubra-nsw-2035', 47, Decimal('0.336')),
 ('camperdown-nsw-2050', 47, Decimal('0.392')),
 ('botany-nsw-2019', 46, Decimal('0.495')),
 ('eastwood-nsw-2122', 44, Decimal('0.772')),
 ('doncaster-vic-3108', 44, Decimal('0.423')),
 ('thornbury-vic-3071', 43, Decimal('0.518')),
 ('abbotsford-vic-3067', 42, Decimal('0.462')),
 ('northmead-nsw-2152', 42, Decimal('0.575')),
 ('ringwood-east-vic-3135', 42, Decimal('0.913')),
 ('burwood-nsw-2134', 42, Decimal('0.396')),
 ('south-brisbane-qld-4101', 42, Decimal('0.318')),
 ('pascoe-vale-vic-3044', 40, Decimal('0.656')),
 ('toowong-qld-4066', 40, Decimal('0.303')),
 ('lidcombe-nsw-2141', 40, Decimal('0.909')),
 ('summer-hill-nsw-2130', 39, Decimal('0.600')),
 ('erskineville-nsw-2043', 39, Decimal('0.355')),
 ('wentworthville-nsw-2145', 38, Decimal('0.613')),
 ('auburn-nsw-2144', 38, Decimal('0.355')),
 ('doncaster-east-vic-3109', 38, Decimal('0.458')),
 ('kensington-vic-3031', 38, Decimal('0.633')),
 ('asquith-nsw-2077', 37, Decimal('0.822')),
 ('st-leonards-nsw-2065', 37, Decimal('0.346')),
 ('kensington-nsw-2033', 37, Decimal('0.311')),
 ('scarborough-qld-4020', 36, Decimal('0.383')),
 ('chippendale-nsw-2008', 36, Decimal('0.514')),
 ('carlton-vic-3053', 36, Decimal('0.350')),
 ('kew-vic-3101', 34, Decimal('0.667')),
 ('hillsdale-nsw-2036', 34, Decimal('0.773')),
 ('heidelberg-vic-3084', 34, Decimal('0.667')),
 ('manly-vale-nsw-2093', 33, Decimal('0.367')),
 ('moonee-ponds-vic-3039', 33, Decimal('0.465')),
 ('hamilton-qld-4007', 33, Decimal('0.363')),
 ('maribyrnong-vic-3032', 32, Decimal('0.410')),
 ('north-strathfield-nsw-2137', 32, Decimal('0

We get 123 affordable suburbs, each of which has at least 30% apartments affordable.

### 2. Distance to CBD

<img src='../charts/apt_dis.png'>

*Source: Domain, World Bank*

Then we take 15km from CBD as a reasonable measure at this stage. Different from houses, apartments' location is more important.

In [8]:
%%sql 
select suburb_id, ttl_apt_tier1, apt_affordability_tier1, distance_km
from combined_suburbs
where apt_affordability_tier1 >= 0.3 and ttl_apt_tier1 >=20 and distance_km <= 15
order by distance_km

 * postgresql://postgres:***@this_postgres
92 rows affected.


[('melbourne-vic-3000', 227, Decimal('0.374'), Decimal('0.000')),
 ('brisbane-city-qld-4000', 86, Decimal('0.305'), Decimal('0.000')),
 ('southbank-vic-3006', 174, Decimal('0.489'), Decimal('1.101')),
 ('south-brisbane-qld-4101', 42, Decimal('0.318'), Decimal('1.167')),
 ('ultimo-nsw-2007', 23, Decimal('0.511'), Decimal('1.661')),
 ('west-end-qld-4101', 83, Decimal('0.432'), Decimal('2.230')),
 ('chippendale-nsw-2008', 36, Decimal('0.514'), Decimal('2.236')),
 ('new-farm-qld-4005', 62, Decimal('0.373'), Decimal('2.305')),
 ('glebe-nsw-2037', 24, Decimal('0.369'), Decimal('2.450')),
 ('teneriffe-qld-4005', 68, Decimal('0.557'), Decimal('2.764')),
 ('richmond-vic-3121', 115, Decimal('0.504'), Decimal('2.824')),
 ('carlton-vic-3053', 36, Decimal('0.350'), Decimal('3.065')),
 ('docklands-vic-3008', 115, Decimal('0.458'), Decimal('3.121')),
 ('newstead-qld-4006', 54, Decimal('0.478'), Decimal('3.166')),
 ('collingwood-vic-3066', 57, Decimal('0.514'), Decimal('3.190')),
 ('camperdown-nsw-2050', 47, Decimal('0.392'), Decimal('3.412')),
 ('subiaco-wa-6008', 26, Decimal('0.520'), Decimal('3.462')),
 ('waterloo-nsw-2017', 94, Decimal('0.465'), Decimal('3.612')),
 ('abbotsford-vic-3067', 42, Decimal('0.462'), Decimal('3.854')),
 ('toowong-qld-4066', 40, Decimal('0.303'), Decimal('3.964')),
 ('north-melbourne-vic-3051', 62, Decimal('0.596'), Decimal('4.021')),
 ('erskineville-nsw-2043', 39, Decimal('0.355'), Decimal('4.402')),
 ('zetland-nsw-2017', 64, Decimal('0.403'), Decimal('4.423')),
 ('newtown-nsw-2042', 27, Decimal('0.386'), Decimal('4.486')),
 ('crows-nest-nsw-2065', 29, Decimal('0.518'), Decimal('4.748')),
 ('alexandria-nsw-2015', 74, Decimal('0.510'), Decimal('4.777')),
 ('stanmore-nsw-2048', 24, Decimal('0.686'), Decimal('5.016')),
 ('fitzroy-north-vic-3068', 25, Decimal('0.510'), Decimal('5.039')),
 ('kensington-nsw-2033', 37, Decimal('0.311'), Decimal('5.162')),
 ('hawthorn-vic-3122', 85, Decimal('0.515'), Decimal('5.185')),
 ('leichhardt-nsw-2040', 21, Decimal('0.488'), Decimal('5.209')),
 ('st-leonards-nsw-2065', 37, Decimal('0.346'), Decimal('5.312')),
 ('kensington-vic-3031', 38, Decimal('0.633'), Decimal('5.431')),
 ('lutwyche-qld-4030', 26, Decimal('0.419'), Decimal('5.546')),
 ('rosebery-nsw-2018', 56, Decimal('0.459'), Decimal('5.650')),
 ('hamilton-qld-4007', 33, Decimal('0.363'), Decimal('5.693')),
 ('indooroopilly-qld-4068', 32, Decimal('0.311'), Decimal('5.724')),
 ('petersham-nsw-2049', 26, Decimal('0.510'), Decimal('5.825')),
 ('kew-vic-3101', 34, Decimal('0.667'), Decimal('5.969')),
 ('brunswick-east-vic-3057', 71, Decimal('0.617'), Decimal('6.574')),
 ('northcote-vic-3070', 56, Decimal('0.523'), Decimal('6.609')),
 ('kingsford-nsw-2032', 23, Decimal('0.390'), Decimal('6.667')),
 ('brunswick-vic-3056', 94, Decimal('0.534'), Decimal('6.788')),
 ('hawthorn-east-vic-3123', 53, Decimal('0.631'), Decimal('6.903')),
 ('lane-cove-nsw-2066', 137, Decimal('0.489'), Decimal('6.927')),
 ('eastlakes-nsw-2018', 24, Decimal('0.889'), Decimal('7.022')),
 ('ascot-vale-vic-3032', 22, Decimal('0.367'), Decimal('7.165')),
 ('summer-hill-nsw-2130', 39, Decimal('0.600'), Decimal('7.294')),
 ('brunswick-west-vic-3055', 47, Decimal('0.475'), Decimal('7.516')),
 ('yarraville-vic-3013', 28, Decimal('0.683'), Decimal('7.904')),
 ('ashfield-nsw-2131', 163, Decimal('0.674'), Decimal('7.979')),
 ('thornbury-vic-3071', 43, Decimal('0.518'), Decimal('8.243')),
 ('moonee-ponds-vic-3039', 33, Decimal('0.465'), Decimal('8.264')),
 ('williamstown-vic-3016', 24, Decimal('0.585'), Decimal('8.563')),
 ('gladesville-nsw-2111', 79, Decimal('0.725'), Decimal('8.907')),
 ('mascot-nsw-2020', 48, Decimal('0.505'), Decimal('9.076')),
 ('botany-nsw-2019', 46, Decimal('0.495'), Decimal('9.080')),
 ('coburg-vic-3058', 27, Decimal('0.435'), Decimal('9.195')),
 ('camberwell-vic-3124', 22, Decimal('0.468'), Decimal('9.217')),
 ('maroubra-nsw-2035', 47, Decimal('0.336'), Decimal('9.244')),
 ('ivanhoe-vic-3079', 32, Decimal('0.492'), Decimal('9.283')),
 ('maribyr

We get 92 suburbs within 15km in total.

### 3. Investment by Median Price Growth

As for the gloomy apartment market from 2017, the first thing we do to help apartment investors is to avoid a big loss. As an investment, we think the annual loss within 5% could be acceptable. So we filter the annual growth under -5% in 2018, 2019 for the policy intervention and 2020, 2021 for the Covid virus impact. 

In [9]:
%%sql 
select suburb_id, apt_5yr_growth, apt_2017_growth, apt_2018_growth, apt_2019_growth, apt_2020_growth, apt_2021_growth, price_apt_2016, price_apt_2017, price_apt_2018, price_apt_2019, price_apt_2020, price_apt_2021
    , ttl_apt_tier1, apt_affordability_tier1, distance_km
from combined_suburbs
where apt_affordability_tier1 >= 0.3 and ttl_apt_tier1 >=20 and distance_km <= 15 and apt_2018_growth >= -0.05 and apt_2019_growth >= -0.05 and apt_2020_growth >= -0.05 and apt_2021_growth >= -0.05
order by apt_5yr_growth desc

 * postgresql://postgres:***@this_postgres
31 rows affected.


[('fitzroy-north-vic-3068', Decimal('0.446'), Decimal('0.196'), Decimal('-0.004'), Decimal('0.040'), Decimal('0.018'), Decimal('0.147'), Decimal('460000'), Decimal('550000'), Decimal('547999'), Decimal('570000'), Decimal('580000'), Decimal('665000'), 25, Decimal('0.510'), Decimal('5.039')),
 ('lutwyche-qld-4030', Decimal('0.446'), Decimal('-0.097'), Decimal('0.168'), Decimal('0.013'), Decimal('0.013'), Decimal('0.338'), Decimal('370000'), Decimal('334000'), Decimal('390000'), Decimal('395000'), Decimal('400000'), Decimal('535000'), 26, Decimal('0.419'), Decimal('5.546')),
 ('reservoir-vic-3073', Decimal('0.395'), Decimal('0.156'), Decimal('0.060'), Decimal('-0.038'), Decimal('0.094'), Decimal('0.082'), Decimal('405000'), Decimal('468000'), Decimal('496000'), Decimal('477000'), Decimal('522000'), Decimal('565000'), 111, Decimal('0.603'), Decimal('13.140')),
 ('pascoe-vale-vic-3044', Decimal('0.394'), Decimal('0.040'), Decimal('0.117'), Decimal('0.010'), Decimal('0.019'), Decimal('0.167'), Decimal('452000'), Decimal('470000'), Decimal('525000'), Decimal('530000'), Decimal('540000'), Decimal('630000'), 40, Decimal('0.656'), Decimal('11.824')),
 ('glenroy-vic-3046', Decimal('0.372'), Decimal('0.064'), Decimal('0.133'), Decimal('0.018'), Decimal('0.087'), Decimal('0.029'), Decimal('390000'), Decimal('415000'), Decimal('470000'), Decimal('478250'), Decimal('520000'), Decimal('535000'), 58, Decimal('0.690'), Decimal('14.214')),
 ('northcote-vic-3070', Decimal('0.343'), Decimal('-0.024'), Decimal('0.147'), Decimal('-0.019'), Decimal('0.107'), Decimal('0.105'), Decimal('469000'), Decimal('457750'), Decimal('525000'), Decimal('515000'), Decimal('570000'), Decimal('630000'), 56, Decimal('0.523'), Decimal('6.609')),
 ('brunswick-west-vic-3055', Decimal('0.306'), Decimal('0.132'), Decimal('0.006'), Decimal('0.037'), Decimal('0.094'), Decimal('0.011'), Decimal('360000'), Decimal('407500'), Decimal('410000'), Decimal('425000'), Decimal('465000'), Decimal('470000'), 47, Decimal('0.475'), Decimal('7.516')),
 ('thornbury-vic-3071', Decimal('0.301'), Decimal('0.032'), Decimal('0.142'), Decimal('-0.018'), Decimal('0.072'), Decimal('0.048'), Decimal('419000'), Decimal('432500'), Decimal('494000'), Decimal('485000'), Decimal('520000'), Decimal('545000'), 43, Decimal('0.518'), Decimal('8.243')),
 ('maribyrnong-vic-3032', Decimal('0.247'), Decimal('0.148'), Decimal('0.007'), Decimal('-0.034'), Decimal('0.070'), Decimal('0.043'), Decimal('385000'), Decimal('442000'), Decimal('445000'), Decimal('430000'), Decimal('460000'), Decimal('480000'), 32, Decimal('0.410'), Decimal('9.590')),
 ('north-ryde-nsw-2113', Decimal('0.216'), Decimal('0.000'), Decimal('0.043'), Decimal('0.033'), Decimal('0.072'), Decimal('0.052'), Decimal('580000'), Decimal('580000'), Decimal('605000'), Decimal('625000'), Decimal('670000'), Decimal('705000'), 27, Decimal('0.551'), Decimal('10.630')),
 ('brunswick-east-vic-3057', Decimal('0.200'), Decimal('0.025'), Decimal('-0.032'), Decimal('0.099'), Decimal('0.016'), Decimal('0.082'), Decimal('458500'), Decimal('470000'), Decimal('455000'), Decimal('500000'), Decimal('508200'), Decimal('550000'), 71, Decimal('0.617'), Decimal('6.574')),
 ('preston-vic-3072', Decimal('0.172'), Decimal('0.030'), Decimal('0.023'), Decimal('0.022'), Decimal('0.055'), Decimal('0.031'), Decimal('422500'), Decimal('435000'), Decimal('445000'), Decimal('455000'), Decimal('480000'), Decimal('495000'), 66, Decimal('0.458'), Decimal('10.065')),
 ('essendon-vic-3040', Decimal('0.159'), Decimal('0.011'), Decimal('0.004'), Decimal('0.023'), Decimal('0.093'), Decimal('0.020'), Decimal('440000'), Decimal('445000'), Decimal('447000'), Decimal('457500'), Decimal('500000'), Decimal('510000'), 57, Decimal('0.460'), Decimal('10.132')),
 ('brunswick-vic-3056', Decimal('0.147'), Decimal('-0.019'), Decimal('0.054'), Decimal('0.041'), Decimal('0.040'), Decimal('0.025'), Decimal('469000'), Decimal('460000'), Decimal('485000'), Decimal('505000'), Decimal('525000'), Decimal('5380

We get 30 suburbs left.

Next, We still seek a minimum 30% capital growth in the past 5 years for the apartment market.

In [10]:
%%sql 
select suburb_id, apt_5yr_growth, apt_2017_growth, apt_2018_growth, apt_2019_growth, apt_2020_growth, apt_2021_growth, price_apt_2016, price_apt_2017, price_apt_2018, price_apt_2019, price_apt_2020, price_apt_2021
    , ttl_apt_tier1, apt_affordability_tier1, distance_km
from combined_suburbs
where apt_affordability_tier1 >= 0.3 and ttl_apt_tier1 >=20 and distance_km <= 15 and apt_2018_growth >= -0.05 and apt_2019_growth >= -0.05 and apt_2020_growth >= -0.05 and apt_2021_growth >= -0.05 and apt_5yr_growth >= 0.3
order by apt_5yr_growth desc

 * postgresql://postgres:***@this_postgres
8 rows affected.


[('lutwyche-qld-4030', Decimal('0.446'), Decimal('-0.097'), Decimal('0.168'), Decimal('0.013'), Decimal('0.013'), Decimal('0.338'), Decimal('370000'), Decimal('334000'), Decimal('390000'), Decimal('395000'), Decimal('400000'), Decimal('535000'), 26, Decimal('0.419'), Decimal('5.546')),
 ('fitzroy-north-vic-3068', Decimal('0.446'), Decimal('0.196'), Decimal('-0.004'), Decimal('0.040'), Decimal('0.018'), Decimal('0.147'), Decimal('460000'), Decimal('550000'), Decimal('547999'), Decimal('570000'), Decimal('580000'), Decimal('665000'), 25, Decimal('0.510'), Decimal('5.039')),
 ('reservoir-vic-3073', Decimal('0.395'), Decimal('0.156'), Decimal('0.060'), Decimal('-0.038'), Decimal('0.094'), Decimal('0.082'), Decimal('405000'), Decimal('468000'), Decimal('496000'), Decimal('477000'), Decimal('522000'), Decimal('565000'), 111, Decimal('0.603'), Decimal('13.140')),
 ('pascoe-vale-vic-3044', Decimal('0.394'), Decimal('0.040'), Decimal('0.117'), Decimal('0.010'), Decimal('0.019'), Decimal('0.167'), Decimal('452000'), Decimal('470000'), Decimal('525000'), Decimal('530000'), Decimal('540000'), Decimal('630000'), 40, Decimal('0.656'), Decimal('11.824')),
 ('glenroy-vic-3046', Decimal('0.372'), Decimal('0.064'), Decimal('0.133'), Decimal('0.018'), Decimal('0.087'), Decimal('0.029'), Decimal('390000'), Decimal('415000'), Decimal('470000'), Decimal('478250'), Decimal('520000'), Decimal('535000'), 58, Decimal('0.690'), Decimal('14.214')),
 ('northcote-vic-3070', Decimal('0.343'), Decimal('-0.024'), Decimal('0.147'), Decimal('-0.019'), Decimal('0.107'), Decimal('0.105'), Decimal('469000'), Decimal('457750'), Decimal('525000'), Decimal('515000'), Decimal('570000'), Decimal('630000'), 56, Decimal('0.523'), Decimal('6.609')),
 ('brunswick-west-vic-3055', Decimal('0.306'), Decimal('0.132'), Decimal('0.006'), Decimal('0.037'), Decimal('0.094'), Decimal('0.011'), Decimal('360000'), Decimal('407500'), Decimal('410000'), Decimal('425000'), Decimal('465000'), Decimal('470000'), 47, Decimal('0.475'), Decimal('7.516')),
 ('thornbury-vic-3071', Decimal('0.301'), Decimal('0.032'), Decimal('0.142'), Decimal('-0.018'), Decimal('0.072'), Decimal('0.048'), Decimal('419000'), Decimal('432500'), Decimal('494000'), Decimal('485000'), Decimal('520000'), Decimal('545000'), 43, Decimal('0.518'), Decimal('8.243'))]

Now 8 suburbs are left for the investors from 905 suburbs in the top 5 cities. 

We can see 7 of 8 suburbs are in Victoria and the 1 left is in Queensland. But Lutwyche in Queensland has the highest growth with our criteria.

**Result Table**

In [24]:
%%sql 
select c.suburb_id, s.suburb, s.state, s.population, s.median_income
    , apt_5yr_growth, apt_2017_growth, apt_2018_growth, apt_2019_growth, apt_2020_growth, apt_2021_growth, price_apt_2016, price_apt_2017, price_apt_2018, price_apt_2019, price_apt_2020, price_apt_2021
    , ttl_apt_tier1, apt_affordability_tier1, distance_km
from combined_suburbs c
left join suburbs s on c.suburb_id = s.suburb_id
where apt_affordability_tier1 >= 0.3 and ttl_apt_tier1 >=20 and distance_km <= 15 and apt_2018_growth >= -0.05 and apt_2019_growth >= -0.05 and apt_2020_growth >= -0.05 and apt_2021_growth >= -0.05 and apt_5yr_growth >= 0.3
order by apt_5yr_growth desc

 * postgresql://postgres:***@this_postgres
8 rows affected.


[('fitzroy-north-vic-3068', 'Fitzroy North', 'VIC', 12339, 52052, Decimal('0.446'), Decimal('0.196'), Decimal('-0.004'), Decimal('0.040'), Decimal('0.018'), Decimal('0.147'), Decimal('460000'), Decimal('550000'), Decimal('547999'), Decimal('570000'), Decimal('580000'), Decimal('665000'), 25, Decimal('0.510'), Decimal('5.039')),
 ('lutwyche-qld-4030', 'Lutwyche', 'QLD', 3454, 47476, Decimal('0.446'), Decimal('-0.097'), Decimal('0.168'), Decimal('0.013'), Decimal('0.013'), Decimal('0.338'), Decimal('370000'), Decimal('334000'), Decimal('390000'), Decimal('395000'), Decimal('400000'), Decimal('535000'), 26, Decimal('0.419'), Decimal('5.546')),
 ('reservoir-vic-3073', 'Reservoir', 'VIC', 50474, 28132, Decimal('0.395'), Decimal('0.156'), Decimal('0.060'), Decimal('-0.038'), Decimal('0.094'), Decimal('0.082'), Decimal('405000'), Decimal('468000'), Decimal('496000'), Decimal('477000'), Decimal('522000'), Decimal('565000'), 111, Decimal('0.603'), Decimal('13.140')),
 ('pascoe-vale-vic-3044', 'Pascoe Vale', 'VIC', 17051, 38012, Decimal('0.394'), Decimal('0.040'), Decimal('0.117'), Decimal('0.010'), Decimal('0.019'), Decimal('0.167'), Decimal('452000'), Decimal('470000'), Decimal('525000'), Decimal('530000'), Decimal('540000'), Decimal('630000'), 40, Decimal('0.656'), Decimal('11.824')),
 ('glenroy-vic-3046', 'Glenroy', 'VIC', 22245, 28028, Decimal('0.372'), Decimal('0.064'), Decimal('0.133'), Decimal('0.018'), Decimal('0.087'), Decimal('0.029'), Decimal('390000'), Decimal('415000'), Decimal('470000'), Decimal('478250'), Decimal('520000'), Decimal('535000'), 58, Decimal('0.690'), Decimal('14.214')),
 ('northcote-vic-3070', 'Northcote', 'VIC', 24561, 45708, Decimal('0.343'), Decimal('-0.024'), Decimal('0.147'), Decimal('-0.019'), Decimal('0.107'), Decimal('0.105'), Decimal('469000'), Decimal('457750'), Decimal('525000'), Decimal('515000'), Decimal('570000'), Decimal('630000'), 56, Decimal('0.523'), Decimal('6.609')),
 ('brunswick-west-vic-3055', 'Brunswick West', 'VIC', 14159, 38584, Decimal('0.306'), Decimal('0.132'), Decimal('0.006'), Decimal('0.037'), Decimal('0.094'), Decimal('0.011'), Decimal('360000'), Decimal('407500'), Decimal('410000'), Decimal('425000'), Decimal('465000'), Decimal('470000'), 47, Decimal('0.475'), Decimal('7.516')),
 ('thornbury-vic-3071', 'Thornbury', 'VIC', 18568, 40352, Decimal('0.301'), Decimal('0.032'), Decimal('0.142'), Decimal('-0.018'), Decimal('0.072'), Decimal('0.048'), Decimal('419000'), Decimal('432500'), Decimal('494000'), Decimal('485000'), Decimal('520000'), Decimal('545000'), 43, Decimal('0.518'), Decimal('8.243'))]

## Conclusion

Again the result is an example and only provides investors a way to select the suburbs to inspect among hundreds of suburbs. With more features from the database, investors can find out their target suburbs for their investment reference.

# Disclaimer

The sole purpose of this research is to provide as many features as possible about a suburb, so that people can find the most appropriate suburb to invest in according to his own condition. 

[^1] Source: [Geonames](https://www.geonames.org/)

[^2] Source: [ibisworld](https://www.ibisworld.com/au/industry/residential-real-estate-advertising/5512/)